sources/pmm.2.31.0.pmm-server.json

```diff
@@ -17,9 +17,9 @@
               "status": "available",
               "critical": false
             },
-            "1.11.0": {
-              "image_path": "percona/percona-xtradb-cluster-operator:1.11.0",
-              "image_hash": "69501813d433aba1b9bd0babf7b7033000696da2a4c7fc582dac00c79a100c82",
+            "1.11.0-dbaas": {
+              "image_path": "perconalab/percona-xtradb-cluster-operator:1.11.0-custom-120",
+              "image_hash": "3ea2887ac3f9cf92cb0678c28433fee5a086298e122f11c33dc286f3da11c763",
```

Comment:
uhm, I'm not sure that it will work. It might break upgrade of operators.

Reply:
@beatahandzelova could you please test the upgrade from 1.10.0 to this version locally?